### PR TITLE
feat(server/resources): Add endpoints to save albums

### PR DESCRIPTION
**Pull request by RSNara**
PR isn't ready to merge, but I've set up two routes to both fetch and save dropboxLinks.

### CODE REVIEW

**Comment by RSNara** [comment]
Ewww.... coveralls failed. We need to include all files in the coverage report, not just the ones that have tests written for them.

**Comment by mandishshah** [comment]
Stupid coverage reports LOL

File: server/resources/users/me/albums.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+export function saveAlbum(request, response) {
+  const doc = Object.assign(request.body, {
+    facebookUserId: request.user.id,
+  });
+
+  Album
+    .create(doc)
+    .then(DB.toObject)
```
Comment (mandishshah) [inline server/resources/users/me/albums.js]: The young Ramba. Jeez! Do we really need this? :$ I will have to look into it that's why :p

**Comment by RSNara** [comment]
Sweet! CI failed.

**Comment by mandishshah** [comment]
Any update on this?

**Comment by RSNara** [comment]
Yeah, the contents from this PR will be inside PR#12

**Pull request closed without merging**